src/App.vue

```diff
@@ -24,7 +24,7 @@ export default {
 @tailwind components;
 
 @import './assets/css/fonts.css';
-@import './assets/css/colors.css';
+/* @import './assets/css/colors.css'; */
```

Comment:
```suggestion
/* @import './assets/css/colors.css'; */
```
This comment could get removed since no longer needed?